src/primitives/transaction.h

```diff
@@ -99,22 +99,34 @@ static inline size_t JOINSPLIT_SIZE(int transactionVersion) {
 class SpendDescriptionV5
 {
 public:
-    uint256 cv;                    //!< A value commitment to the value of the input note.
-    uint256 nullifier;             //!< The nullifier of the input note.
-    uint256 rk;                    //!< The randomized public key for spendAuthSig.
+    uint256 inner_cv;                    //!< A value commitment to the value of the input note.
+    uint256 inner_nullifier;             //!< The nullifier of the input note.
+    uint256 inner_rk;                    //!< The randomized public key for spendAuthSig.
```

Comment:
Why do the `inner_*` fields still need to be public?

Reply:
The entire class is going to go away, so I wasn't particularly focused on that.